theme/style.css

```diff
@@ -16,3 +16,28 @@
 }
 
 /* style your grid sections here!  */
+
+.ashlea {
+  display: flex;
+  background-color: purple;
+  color: white;
+  position: relative;
+  font-family: 'oswald';
+  font-weight: bold;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  }
+
+.list {
+  list-style: none;
+}
+
+.list a:hover {
+  background-color: yellow;
+}
+
+ul li a {
```

Comment:
This rule is probably a bit too broad... there are going to be a lot of anchor tags in list items on this page, and not all of them should be green 😬